### ISSUE TITLE
SEGSAdmin Database Setup hanging

### ISSUE THREAD

**Issue by Magentah** [Segs/Segs#528]
### Expected behavior
settings.cfg is created successfully. Databases are created successfully.

### Actual behavior/symptoms
settings.cfg doesn't get created, although SEGSAdmin says it has been created.
Create New Databases hangs on Checking for existing databases.

```
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
QMetaObject::connectSlotsByName: No matching signal for on_add_user()
QMetaObject::connectSlotsByName: No matching signal for on_add_admin_user()
QMetaObject::connectSlotsByName: No matching signal for on_generate_config_file()
Checking for existing databases...
Api retrieval and stored success
"https://github.com/Segs/Segs/releases/tag/v0.5.0"
CURRENT VERSION
Checking for existing databases...
Setting arguments...
Failed to start DBTool Add User...
Checking for existing databases...
```

It worked fine after copying the template and manually changing the IPs, then running dbtool manually.

### Steps to reproduce behavior/symptoms
Run SEGSAdmin and attempt to Generate Config Files and Create New Databases.

### SEGS version

0.5.0

### Operating system
Ubuntu 18.04
Wine 3.0.2

**Comment by miles200** [comment]
Will take a look

**Comment by broxen** [comment]
The only thing I could find for `QStandardPaths: XDG_RUNTIME_DIR not set...` relates to running as Root, but it seems that wasn't the case here, so I'm not sure what could cause this.

**Comment by Magentah** [comment]
As a note, I was running with sudo, so it may be related.

**Comment by broxen** [comment]
Ahh, in that case it could be as simple as that.

Maybe we can check for this error and warn the user to run SEGSAdmin without sudo privileges

**Comment by miles200** [comment]
@broxen Before I go trawling the interwebs, any idea on how I'd check for elevated privileges when using SEGSAdmin?

**Comment by broxen** [comment]
I was actually thinking of the opposite direction. If we get this error, we assume the user is running with sudo and give them a message

**Comment by broxen** [comment]
@miles200 I think this is fixed in your latest PR, no?

**Comment by miles200** [comment]
No sorry not looked at this one yet

**Comment by miles200** [comment]
@broxen is this just a case of adding a message to the below: 

```
CURRENT VERSION
Checking for existing databases...
Setting arguments...
Failed to start DBTool Add User... **Please ensure you are not running as an adminstrator or sudo user**
Checking for existing databases...
```

**Comment by broxen** [comment]
Yep. I'd maybe emphasis `NOT` since I had to read it a couple times to catch that. But I think other than that it's good
